Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM renovate/ruby@sha256:1c25e2b8e76557412434dd4c6a1c6057aafeae4f26418d119cf388bca265b295
+# renovate: datasource=rubygems depName=cocoapods versioning=ruby
+ARG COCOAPODS_VERSION=1.9.2
+
+FROM renovate/buildpack:2-ruby@sha256:c83d691fae1ad0d4594c22607d594cad152bdfdf436998268bb57a9b4329a890
+
+ARG RUBY_VERSION=2.7.1
```

Comment:
It will be interesting to see if Cocoapods 1.0.0 works with Ruby 2.7.1. If not then we should lower the Ruby version temporarily before raising it again.